src/Codeception/Util/ReflectionHelper.php

```diff
@@ -85,7 +85,7 @@ public static function getClassFromParameter(ReflectionParameter $parameter)
         }
 
         $type = $parameter->getType();
-        if ($type === null) {
+        if ($type === null || $type->isBuiltin()) {
```

Comment:
Cool, there is a method for types, I thought that it may be necessary to hardcode a list.